website/docs/r/oauth_client.html.markdown

```diff
@@ -90,7 +90,7 @@ Link.
 * `service_provider` - (Required) The VCS provider being connected with. Valid
   options are `ado_server`, `ado_services`, `bitbucket_data_center`, `bitbucket_hosted`, `bitbucket_server`(deprecated), `github`, `github_enterprise`, `gitlab_hosted`,
   `gitlab_community_edition`, or `gitlab_enterprise_edition`.
-* `agent_pool_id` - (Optional) An existing Agent pool id within the organization which has Private VCS support enabled via Premium SKU.
+* `agent_pool_id` - (Optional) An existing agent pool id within the organization that has Private VCS support enabled.
```

Comment:
@paulanunda I'd prob also put `id` in all caps `ID`

Reply:
Good call; fixed: ed2fedab5ab39e7baa1fb0098644724d637c36b2